config/api_router.py

```diff
@@ -18,12 +19,13 @@
 
 router.register("users", UserViewSet)
 router.register("opportunity", OpportunityViewSet, basename="Opportunity")
-router.register("opportunity/(?P<opportunity_id>.+)/deliver_visit", UserVisitViewSet, basename="UserVisit")
+router.register("opportunity/(?P<opportunity_id>.+)/user_visit", UserVisitViewSet, basename="UserVisit")
```

Comment:
nit: Can we remove this API endpoint now since the data from this is also include in the deliver_progress endpoint?

Reply:
I left it intentionally since it is a much more efficient API if you just want visit info, and there isn't really a cost, but happy to remove it if you feel strongly.

Reply:
I am fine with keeping this API. Just wanted to get your thoughts about keeping this.